WordPress/Classes/Extensions/UINavigationController+Helpers.swift

```diff
@@ -33,4 +33,17 @@ extension UINavigationController {
             }
         }
     }
+
+    /// Fixes a crash in iOS 13 (#12882) where presenting a UIDocumentMenuViewController in a webView
+    /// doesn't rautomattically ecognize the location for presenting the menu hence the crash.
+    /// The warning is probably a bug on iOS or on WebKit since replacing with UIDocumentPickerViewController doesn't prevent the crash.
+    ///
+    @objc override open func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        if #available(iOS 13, *), UIDevice.current.userInterfaceIdiom == .phone,
+            let webKitVC = topViewController as? WebKitViewController,
+            viewControllerToPresent is UIDocumentMenuViewController {
```

Comment:
Do we really need this `is UIDocumentMenuViewController`? I tried without it and the result was the same.

That way we can get rid of the warning.

Reply:
I guess I was concerned about presenting anything else that we might not want to present from the location of the tap. Since I am checking for WebKitVC at the top of the presentation stack and I don't believe we would want to display anything other than a popover on top of that. I removed the check for UIDocumentMenuViewController because the warning is annoying and it makes sense